packages/dashboard/src/components/defaultViewport/constants.ts

```diff
@@ -0,0 +1,34 @@
+import { DateRangePickerProps } from '@cloudscape-design/components';
+
+export const messages = {
+  title: 'Default viewport',
+  placeholder: 'Default viewport',
+  todayAriaLabel: 'Today',
+  nextMonthAriaLabel: 'Next month',
+  previousMonthAriaLabel: 'Previous month',
+  customRelativeRangeDurationLabel: 'Duration',
+  customRelativeRangeDurationPlaceholder: 'Enter duration',
+  customRelativeRangeOptionLabel: 'Custom range',
+  customRelativeRangeOptionDescription: 'Set a custom relative range',
+  customRelativeRangeUnitLabel: 'Unit of time',
+  dateTimeConstraintText:
+    'For date, use YYYY/MM/DD. For time, use 24 hr format.',
+  relativeModeTitle: 'Relative range',
+  absoluteModeTitle: 'Absolute range',
+  relativeRangeSelectionHeading: 'Choose a range',
+  startDateLabel: 'Start date',
+  endDateLabel: 'End date',
+  startTimeLabel: 'Start time',
+  endTimeLabel: 'End time',
+  clearButtonLabel: 'Clear and dismiss',
+  cancelButtonLabel: 'Cancel',
+  applyButtonLabel: 'Apply',
+  formatRelativeRange: (e: DateRangePickerProps.RelativeValue) =>
+    e.amount === 1 ? `Last ${e.unit}` : `Last ${e.amount} ${e.unit}s`,
+  formatUnit: (e: DateRangePickerProps.TimeUnit, n: number) =>
+    1 === n ? e : `${e}s`,
+  dateRangeIncompleteError:
+    'The selected date range is incomplete. Select a start and end date for the date range.',
+  dateRangeInvalidError:
+    'The selected date range is invalid. The start date must be before the end date.',
+};
```

Comment:
Do we need to do this if we can use this? https://cloudscape.design/get-started/dev-guides/internationalization/

Also, is this how react-intl handles internationalization?

Reply:
dashboard doesn't have `react-intl` support yet 🤔 can definitely bring in the default cloudscape messages for this though!